### PR TITLE
[Impeller] add drawVertices and drawAtlas benchmarks.

### PR DESCRIPTION
**Pull request by jonahwilliams**
Adds impeller benchmarks to track the progress of improvements made in https://github.com/flutter/flutter/issues/131345

### CODE REVIEW

**Comment by jonahwilliams** [comment]
This adds the benchmarks but not the CI configuration, holding off until https://github.com/flutter/flutter/pull/131837 lands too

**Review by bdero — APPROVED** [review]
RSLGTM

**Review by gaaclarke — APPROVED** [review]
LGTM, make sure when you hook it into CI we get readings for vulkan, opengles, and metal.

**Pull request merged**
